src/nanoemoji/nanoemoji.py

```diff
@@ -80,19 +80,38 @@ class InputGlyph(NamedTuple):
 class ColorGenerator(NamedTuple):
     apply_ufo: Callable[[ufoLib2.Font, Sequence[ColorGlyph]], None]
     apply_ttfont: Callable[[ufoLib2.Font, Sequence[ColorGlyph], ttLib.TTFont], None]
+    font_ext: str  # extension for font binary, .ttf or .otf
 
 
 _COLOR_FORMAT_GENERATORS = {
-    "glyf": ColorGenerator(lambda *args: _glyf_ufo(*args), lambda *_: None),
-    "colr_0": ColorGenerator(lambda *args: _colr_ufo(0, *args), lambda *_: None),
-    "colr_1": ColorGenerator(lambda *args: _colr_ufo(1, *args), lambda *_: None),
-    "svg": ColorGenerator(lambda *_: None, lambda *args: _svg_ttfont(*args, zip=False)),
-    "svgz": ColorGenerator(lambda *_: None, lambda *args: _svg_ttfont(*args, zip=True)),
+    "glyf": ColorGenerator(lambda *args: _glyf_ufo(*args), lambda *_: None, ".ttf"),
+    "glyf_colr_0": ColorGenerator(
+        lambda *args: _colr_ufo(0, *args), lambda *_: None, ".ttf"
+    ),
+    "glyf_colr_1": ColorGenerator(
+        lambda *args: _colr_ufo(1, *args), lambda *_: None, ".ttf"
+    ),
+    "cff_colr_0": ColorGenerator(
```

Comment:
if we are adding the prefix `cff` to the --color_format, maybe we can get rid of the extra `--cff_version` option, and select the right CFF table version based on the --color_format prefix `cff_colr_1` vs `cff2_colr_1`

Reply:
Done